internal/mempool/mempool.go

```diff
@@ -68,7 +68,7 @@ const (
 	// physical space available for TSpend votes and thus is a hard limit.
 	MempoolMaxConcurrentTSpends = 7
 
-	// unknownBlockHeight represents a value indicating that best known block
+	// blockHeightUnknown represents a value indicating that best known block
```

Comment:
Same case here and the others.  Please squash these fixes into the earlier commit(s) that introduce the original code that is being updated by this commit.